common/archive.cpp

```diff
@@ -80,6 +83,10 @@ int Archive::listMatchingMembers(ArchiveMemberList &list, const Path &pattern, b
 	return matches;
 }
 
+SeekableReadStream *Archive::createReadStreamForMemberAltStream(const Path &path, AltStreamType altStreamType) const {
+	return nullptr;
```

Comment:
Should this try to return something like the  `Archive` implementation of the regular read stream?

Reply:
No, because `createReadStreamForMember` returns the primary stream, I don't think there's any situation where we'd want to open the primary stream when an alternate stream was requested.